sys-block/open-iscsi/files/iscsid.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Open-iSCSI
+Documentation=man:iscsid(8) man:iscsiuio(8) man:iscsiadm(8)
+After=network.target NetworkManager-wait-online.service iscsiuio.service tgtd.service targetcli.service
```

Comment:
can we get rid of `NetworkManager-wait-online.service` here? does open-iscsi ship tgtd or targetcli?